app/routes/api+/model-filters.ts

```diff
@@ -68,11 +72,23 @@ export async function loader({ context, request }: LoaderFunctionArgs) {
     const { name, queryKey, queryValue, selectedValues, ...filters } =
       parseData(searchParams, ModelFiltersSchema);
 
-    const where: Record<string, any> = {
+    let where: Record<string, any> = {
       organizationId,
       OR: [{ id: { in: (selectedValues ?? "").split(",") } }],
     };
 
+    const genericKeys = ["deletedAt"];
```

Comment:
I am having trouble understanding this. Can you explain why you are doing it and add some annotations. Then we can maybe see if there is a better way to manage it.